cloudify_cli/tests/commands/test_deployments.py

```diff
@@ -64,22 +64,32 @@ def setUp(self):
     def test_deployment_update_get(self):
```

Comment:
We don't appear to have a test that confirms that a deployment update with preview actually leaves the deployment alone?